python/hsfs/feature_view.py

```diff
@@ -2553,6 +2564,11 @@ def _create_feature_monitoring(
                 Options are "HOURLY", "DAILY", "WEEKLY", "MONTHLY", defaults to "DAILY".
```

Comment:
Same as previous comment.
In the JobScheduler constructor, it seems MONTHLY it is not supported, only HOURLY, DAILY and WEEKLY.